### DISCUSSION TITLE
how to use SimpleSDXL (fooocus ui) google colab? please help

### DISCUSSION THREAD

**Discussion by rafiislambd** [lllyasviel/Fooocus#2243] [General]
how to use SimpleSDXL (fooocus ui) google colab? please help

**Comment by mashb1t** [comment]
For Fooocus: Click the shiny "Open in Colab" button in https://github.com/lllyasviel/Fooocus?tab=readme-ov-file#colab
For SimpleSDXL: no idea, this repo is not associated nor will provide support for this tool. Feel free to check SimpleSDXL documentation/ issues / discussions.

**Reply by metercai** [reply]
@rafiislambd, the same as Fooocus, because of SimpleSDXL is a derivative of Fooocus. any questions, you can ask here: https://github.com/metercai/SimpleSDXL/issues
